### PR TITLE
adds modernizr check

### PR DESCRIPTION
**Pull request by MatthewVita**

### CODE REVIEW

File: client/app/index.js [hunk 1]
```diff
@@ -1,48 +1,64 @@
...
+    Modernizr.fontface &&
+    Modernizr.hsla &&
+    Modernizr.hashchange &&
+    Modernizr.input &&
+    Modernizr.opacity &&
+    Modernizr.rgba &&
+    Modernizr.svg &&
+    Modernizr.svgclippaths &&
+    Modernizr.sessionstorage)) {
+  window.location = 'http://www.old-browser.org/en-us/?referer=' + window.location.host;
```
Comment (MatthewVita) [inline client/app/index.js:14]: @cameronscott137 this is where we cut off b/c the user is clearly using something less than IE9

File: client/app/shared/modernizr/modernizr.js [hunk 2]
```diff
@@ -0,0 +1,1406 @@
+/*!
```
Comment (MatthewVita) [inline client/app/shared/modernizr/modernizr.js:1]: @cameronscott137 I couldn't find a bundled up Modernizr on NPM so I just threw this one into `shared/modernizr/`

**Comment by MatthewVita** [comment]
@cameronscott137 thanks in advance for doing a quick code review.

**Comment by MatthewVita** [comment]
@cameronscott137 `System.out.println("ping");`

**Comment by chadwhitacre** [comment]
I see this applying to the public-facing site. I don't see it applying to the dashboard. Is that right? I think it is because I think we're not using Angular on the dashboard (yet: #342).

I don't have an old IE handy to check the negative case, but the positive case is working for me. Merging!

**Comment by MatthewVita** [comment]
@whit537 thank you for doing this review. I've tested the positive and negative cases on my end sufficiently (was able to roll back multiple Internet Explorer engines via the devtools feature).

I'm not very happy with the fact that I couldn't find a bundled up Modernizr on NPM so I just threw the dependency into our `shared/modernizr/` area. C'est la vie.

Regarding dashboard. I didn't include it there because we're not using Angular over there yet. Also the current Rails dashboard could probably work just fine on ancient browsers xD.

**Pull request merged**